tests/main/interfaces-udisks2/task.yaml

```
@@ -41,7 +41,9 @@ execute: |
 
     device="$(losetup -j "$FS_PATH" | cut -d: -f1)"
 
-    test-snapd-udisks2.udisksctl mount -b "$device" -t ext4 | MATCH "Mounted /dev/"
+    # We retry because there is a race with the device becoming
+    # registered by udisks2. The issue can be easy reproduced in ubuntu-20.04
+    retry-tool -n 3 --wait 1 test-snapd-udisks2.udisksctl mount -b "$device" -t ext4 | MATCH "Mounted /dev/"
```

This use of retry-tool is incorrect. The MATCH will be invoked on the output of retry-tool, not mount.  Please wrap it in sh -c: 
```suggestion
    retry-tool -n 3 --wait 1 sh -c "test-snapd-udisks2.udisksctl mount -b \"$device\" -t ext4 | MATCH 'Mounted /dev/'"
```